test/groovy/TestsPublishResultsTest.groovy

```diff
@@ -0,0 +1,112 @@
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertTrue
+
+import util.Rules
+
+class TestsPublishResultsTest extends BasePipelineTest {
+    Map publisherStepOptions
+    List archiveStepPatterns
+
+    @Rule
+    public RuleChain ruleChain = RuleChain.outerRule(Rules.getCommonRules(this))
+
+    def stepUnderTest
```

Comment:
please rename to `testsPublishResultsScript.groovy` in order to keep consistency across tests